test/cypress/integration/pages/repay-student-debt.js

```diff
@@ -7,25 +7,25 @@ describe( 'Paying for College', () => {
     it( 'should display Direct debit and extra payments', () => {
       page.open();
       page.click( 'Federal' );
-      page.selectQuestion ( '2' );
+      page.selectQuestion( '2' );
```

Comment:
I don’t believe the government curtain hurt me I would hope not because I love my country